test/code/providers/Container_ImageInventory_Class_Provider_UnitTest.cpp

```diff
@@ -93,30 +105,62 @@ class ImageInventoryTest : public CppUnit::TestFixture
 		StandardTestEnumerateInstances<mi::Container_ImageInventory_Class_Provider>(m_keyNames, context, CALL_LOCATION(errMsg));
 
 		// Get images using command line
-		CPPUNIT_ASSERT(!system("docker images -q --no-trunc > /tmp/docker_image_ids.txt"));
+		CPPUNIT_ASSERT(!system("docker images --no-trunc > /tmp/docker_image_ids.txt"));
 
 		FILE* idFile = fopen("/tmp/docker_image_ids.txt", "r");
 		CPPUNIT_ASSERT(idFile);
 
-		wchar_t id[65];
+		wchar_t line[200];
 		set<wstring> allIds;
 
-		// Full image IDs (one per line)
-		while (fwscanf(idFile, L"%ls", id) != EOF)
+		int matched = 0;
+
+		// Full image info (header plus one per line)
+		if (fgetws(line, 200, idFile) != NULL)
```

Comment:
200 is a magic number, you know what I want from prior ode reviews. Better to use C++ String class, at which point this doesn't matter at all. We have infrastructure to make that easy.